src/main/java/io/stargate/sgv2/jsonapi/service/resolver/AlterTableCommandResolver.java

```diff
@@ -91,6 +91,13 @@ private List<AlterTableAttempt> handleAddColumns(
       TableSchemaObject tableSchemaObject,
       AlterTableOperationImpl.AddColumns addColumnsOperation) {
 
+    // 1. "add":{}
```

Comment:
+1 for checking like this, but we should then return an error if no attempts were created in the resolveTableCommand() function. 

code is probably a new MISSING_ALTER_TABLE_OPERATIONS and we can re-use for all the diff alter table types

Reply:
solved, thanks